compiler/rustc_mir/src/transform/check_consts/validation.rs

```diff
@@ -874,13 +874,13 @@ impl Visitor<'tcx> for Validator<'mir, 'tcx> {
             }
 
             TerminatorKind::InlineAsm { .. } => self.check_op(ops::InlineAsm),
-            TerminatorKind::Abort => self.check_op(ops::Abort),
 
             TerminatorKind::GeneratorDrop | TerminatorKind::Yield { .. } => {
                 self.check_op(ops::Generator(hir::GeneratorKind::Gen))
             }
 
-            TerminatorKind::Assert { .. }
+            TerminatorKind::Abort
```

Comment:
Maybe add a comment here saying that `Abort` terminators are only used for `#[unwind(aborts)]` and since CTFE does not unwind, they are thus unreachable?

Reply:
I could just `span_bug` here actually. AFAICT, there are only `Abort`s in "cleanup" blocks.

Reply:
That sounds like a good idea but does not replace a comment. :)